integration_test/test/code_generation/app_with_defaults_test.exs

```diff
@@ -161,4 +161,29 @@ defmodule Phoenix.Integration.CodeGeneration.AppWithDefaultsTest do
       end)
     end
   end
+
+  describe "phx.gen.auth" do
+    test "has no compilation or formatter warnings" do
+      with_installer_tmp("new with defaults", fn tmp_dir ->
+        {app_root_path, _} = generate_phoenix_app(tmp_dir, "phx_blog")
+
+        mix_run!(~w(phx.gen.auth Accounts User users), app_root_path)
+
+        assert_no_compilation_warnings(app_root_path)
+        assert_passes_formatter_check(app_root_path)
+      end)
+    end
+  end
+
+  @tag database: :postgresql
+  test "has a passing test suite" do
```

Comment:
Should this be moved inside the describe block above?

Reply:
The same applies to the test files, btw. :)

Reply:
Thanks for catching that. I fixed them all in 02e8988ebff9e022cd326d64c98be695b7ba73bb.